dbms/src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -139,9 +139,21 @@ namespace
                     config.keys(database_config, table_names);
 
                     /// Read table properties
-                    for (const String & table_name : table_names)
+                    for (const String & table : table_names)
                     {
-                        const auto filter_config = database_config + "." + table_name + ".filter";
+                        String table_name = table;
```

Comment:
Maybe it would be better to rename `table` to something like `key_in_database_config`. Currently you have two variables `table` and `table_name` in the same scope and it looks confusive.